WhaleSpotting/react-app/src/api/apiClient.ts

```diff
@@ -3,22 +3,22 @@ import authService from "../components/api-authorization/AuthorizeService";
 
 export async function fetchPendingSightings(pageNumber: number): Promise<SightingApiModel[]> {
     const headers = await getHeaders();
-    return await fetch(`https://localhost:5001/sightings/pending?page=${pageNumber}&pageSize=10`, {
+    return await fetch(`https://localhost:5001/api/sightings/pending?page=${pageNumber}&pageSize=10`, {
```

Comment:
as you are changing the others, can you make this
```suggestion
    return await fetch(`api/sightings/pending?page=${pageNumber}&pageSize=10`, {
```

Reply:
:)